quadlet-files/quadlet-demo.yml

```diff
@@ -18,50 +18,50 @@ metadata:
   name: quadlet-demo
 spec:
   containers:
-  - name: wordpress
-    image: docker.io/library/wordpress:4.8-apache
-    env:
-    - name: WORDPRESS_DB_HOST
-      value: quadlet-demo-mysql
-    - name: WORDPRESS_DB_PASSWORD
-      valueFrom:
-        secretKeyRef:
-          name: mysql-root-password-kube
-          key: password
-    volumeMounts:
-    - name: wordpress-persistent-storage
-      mountPath: /var/www/html
-    resources:
-      requests:
-        memory: "64Mi"
-        cpu: "250m"
-      limits:
-        memory: "128Mi"
-        cpu: "500m"
-  - name: envoy
-    image: docker.io/envoyproxy/envoy:v1.25.0
-    volumeMounts:
+    - name: wordpress
```

Comment:
I don't like over indentation of arrays (you can see that the playbook does not do that)

Reply:
Those are changes that ansible-lint aka yamllint proposed. For $REASONS it does not complain for playbooks not indenting like this.

But I can of course just drop the commit, if you wish.

Reply:
I've added a .yamllint file that aligns with my convention and should remove this error. Just rebase

Reply:
> I've added a .yamllint file that aligns with my convention and should remove this error. Just rebase

Thanks, but then the indentation in the playbook is off. I can of course fix that, too. But then it does not look like Ansible playbooks normally do...

Reply:
What command are you running? When I run `ansible-lint playbook.yml` the only indentation error I'm getting is about your addition of the Firewall rules

Reply:
```
$ ansible-lint --offline --profile=production
```

Might be a difference in the version of ansible-lint and yamllint:
```
$ ansible-lint --version
ansible-lint 6.14.2 using ansible 2.14.3
$ yamllint --version
yamllint 1.28.0
$
```

Reply:
Sorry, I forgot to drop the "fix quadlet-demo.yml" commit during the rebase.

Not it looks better. Hold on...